07LessonSeven/homework/css/style.css

```diff
@@ -1,59 +1,82 @@
 body {
-  font-family: sans-serif;
-  font-weight: lighter;
+  font-family: sans-serif, Arial;
+  font-size: 14px;
 }
+
+#container {
+  display: block;
+  width: 100%;
+  height: 100%;
+  margin: 0 auto;
+}
+
 h1 {
-  padding-left: 170px;
+  padding-left: 12.5%;
 }
+
 nav {
-  background-color: #282d2e;
-  height: 80px;
+  height: 60px;
+  padding-top: 16px;
+  margin-bottom: 10px;
 }
-a:link {
-  text-decoration: none;
-  color: white;
+
+nav, #looking {
+  background-color: #313131;
 }
+
 ul {
   list-style-type: none;
-  padding-left: 170px;
-
+  padding-left: 12.5%;
 }
+
 li {
-  float: left;
-  font-size: 16px;
-  padding-right: 25px;
-  padding-top: 28px;
-}
-div {
-  display: flex;
-  flex-flow: row;
-  margin: auto;
-  margin-top: 10px;
-  width: 75%;
-  height: 230px;
+  display: inline-block;
+  margin-right: 15px;
 }
-#employee {
-  background-color: #53c25b;
-  text-align: left;
-  padding: 30px 25px;
+
+a {
+  text-decoration: none;
   color: white;
 }
-#job {
-  float: right;
-  background-color: #282d2e;
-  padding: 30px 25px 30px 25px;
-  text-align: left;
-  color: white;
+
+.hire_looking {
+  display: block;
+  width: 75%;
+  margin: 0 auto;
+}
+
+#hire, #looking {
```

Comment:
Try not to use so many ID selectors. Use classes and element selectors instead. By using the ID selector you are using a selector with a very high specificity (the shark, remember) and it would be very tedious to override its properties with other selectors in the future (would have to use inline css and !important, you don't want that). Always think about making it easier for your future self to modify your code as needed.